testing/trino-tests/src/test/java/io/trino/sql/planner/TestHivePartitionedTpchCostBasedPlan.java

```diff
@@ -16,8 +16,8 @@
 import java.util.stream.Stream;
 
 /**
- * This class tests cost-based optimization rules related to joins. It contains unmodified TPCH queries.
```

Comment:
nit: comment improvements in existing code could be separate commit

Reply:
done